src/main/java/com/kelinci/mainapp/api/UserController.java

```diff
@@ -9,24 +9,18 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
+import java.util.stream.Collectors;
 
 @RestController
 public class UserController {
-    private OurUser registeredUser;
-    private OurUser confirmedUser;
+    private OurUser ourUser;
```

Comment:
Skoro mamy listę userów, to potrzebujemy tego pola? Przecież ten user także powinien się (albo już zawiera się) w liście listOfUser, nie?

Reply:
można go usunąć, ale wtedy nam @GetMappint na /lastuser, który zwraca ourUser będzie nieaktywny - może go usunąć?